STYLEGUIDE.md

```diff
@@ -71,10 +72,11 @@ Must (not), should (not), recommended, suggested, may, and can generally follow
 ## Images/Figures
 
 - Lower case file names and extensions, separate with hyphens, 3-character extensions
-- For ease of visibility, explanatory images should be full width; expository or those with very simple subjects can be half-width, right aligned
+- For ease of visibility, images should be either full width, or 500 px if appropriate
 - Use PNG for all images, except photographs over 100-200 KB as a PNG (use moderately compressed JPEG); avoid sizes above 2000 x 2000 or below 1000 x 1000 if full width, or half of each if half width
+- GIFs should be compressed to 5 frames/s and should target a length of around 5-10 seconds, and be 500 px unlesss needed to illustrate a more complex or important subject
 - Always include alt text describing the content of the image
-- No space before, one space ("|") after if full width
+- No line break before, one line break ("|") after an image/GIF if it is at the end of a section
```

Comment:
I think we have one blank line before and after every image and one line break ("|") followed by three blank lines if it is at the end of a section.

Reply:
Yeah, this is extremely confusingly worded, my fault. I'll rewrite it.

Reply:
Reworded